test/support/spec.rb

```diff
@@ -763,11 +791,11 @@ def at_least(n)
     # FIXME: support endless ranges
     case n
     when :once
-      @count_restriction = 1..9_999_999
+      @count_restriction = 1..
     when :twice
-      @count_restriction = 2..9_999_999
+      @count_restriction = 2..
     else
-      @count_restriction = n..9_999_999
+      @count_restriction = n..
```

Comment:
Those currently cause the CI to fail... I'm going to investigate the issue 😄

Reply:
This seems to be related to NatalieParser. I opened natalie-lang/natalie_parser#4 with more information!